assets/js/modules/analytics/components/setup/SetupForm.js

```diff
@@ -87,6 +105,10 @@ export default function SetupForm( { finishSetup } ) {
 		}
 	}, [ hasEditScope, autoSubmit, submitForm ] );
 
+	if ( ! gtmContainersResolved ) {
+		return <ProgressBar />;
+	}
```

Comment:
Same here. We already have the progress bar in the `SetupMain` component. Please, update that component to keep showing the progress bar if we are not ready to display the setup form.

Reply:
Same here. Read comments from Nahid which make sense to me. Ignore this one.